docs/09_Pages/Navigating_Pages/Internal_Page_on_App_Load.md

```diff
@@ -0,0 +1,37 @@
+The default behavior for DrupalGap is to direct to the front page set in the `settings.js` file. This breaks the sharing feature that users are used to on a typical web site. This code may be added to `settings.js` to enable direction to internal pages.
+
+**This method has only been tested in a web app.**
+
+```
+var landingPage = 'dashboard';
+if(location.hash){
+  // additional logic can be added here for other internal page types and exclude certain pages from being directed to on the initial app load.
+  goToPage = location.hash.replace("#","");
+  landingPage = goToPage.replace('node_', 'node/');
+}
+
+// fix drupalgap_back() to not break if coming from external page
```

Comment:
@mattshoaf Thank you for sharing this workaround. I'd be hesitant to add this to the docs, overwriting DG core isn't my favorite. But you've come up with a clever idea above, I'd like to extract it and make it possible for DG core to route to an internal page (not just the front page).